pkg/common/telemetry/config.go

```diff
@@ -2,20 +2,57 @@ package telemetry
 
 import (
 	"github.com/sirupsen/logrus"
+	"time"
 )
 
 type MetricsConfig struct {
-	FileConfig  FileConfig
-	Logger      logrus.FieldLogger
-	ServiceName string
-	Sinks       []Sink
+	FileConfig       FileConfig
+	Logger           logrus.FieldLogger
+	ServiceName      string
+	Sinks            []Sink
+	TimerGranularity time.Duration
```

Comment:
Seems like both the tests and production default to a millisecond for this. Is there a reason to provide a configurable on a struct v.s. just providing an unexported package-level const to use at this point in time?

Reply:
I don't have any compelling reasons to expose `TimerGranularity` externally at the moment, so this could also be set as a const right now. Probably simpler to go about it that way, and if any specific use cases come up later for needing to configure this, we can always add it back to the `MetricsConfig` struct.